src/libostree/ostree-repo-prune.c

```diff
@@ -160,12 +161,20 @@ _ostree_repo_prune_tmp (OstreeRepo *self,
  * Prune static deltas, if COMMIT is specified then delete static delta files only
  * targeting that commit; otherwise any static delta of non existing commits are
  * deleted.
+ *
+ * This function takes an exclusive lock on the @self repository.
```

Comment:
These docs will show up in non-experimental builds but...eh, it's fine.

Reply:
Well, the lock is still taken when the build is not experimental. Only the APIs aren't exposed on non-experimental builds.